backend/src/main/kotlin/hu/bme/sch/cmsch/component/riddle/RiddleComponent.kt

```diff
@@ -73,6 +81,44 @@ class RiddleComponent(
 
     /// -------------------------------------------------------------------------------------------------------------------
 
+    val shadowBanModerationGroup = SettingProxy(componentSettingService, component,
+        "shadowBanModerationGroup", "", type = SettingType.COMPONENT_GROUP, persist = false,
+        fieldName = "Riddle beadások moderálása - Shadow Ban",
+        description = "Küldjük el pihenni a \"túl aktív\" játékosokat, de csak titokban"
+    )
+
+    val userShadowBanList = SettingProxy(componentSettingService, component,
+        "userShadowBanList", "", serverSideOnly = true, type = SettingType.LONG_TEXT,
+        fieldName = "Tiltott játékosok listája",
+        description = "Ezektől a játékosoktól nem fogadunk el megoldásokat. internalId megadásával lehet egy játékost kitiltani"
```

Comment:
I'd add "separated by commas" or some examples on how to put multiple values into them. Also keep in mind that it might contains line breaks.

Reply:
done